.github/workflows/semgrep.yml

```diff
@@ -0,0 +1,24 @@
+name: "Semgrep"
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+    paths:
+      - .github/workflows/semgrep.yml
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  analyze:
+    name: Scan
+    runs-on: ubuntu-latest
+    if: (github.actor != 'dependabot[bot]')
```

Comment:
can also exclude the renovate bot https://github.com/brave/brave-core/pulls/app%2Frenovate